### PR TITLE
Hello Microverse Project Pull Request

### PR DESCRIPTION
**Pull request by vikas924**
Hello there,

In this project, I have set up a "Hello world" repository. My tasks include:

1.  Documented a professional README.md.
2.  Included a .gitignore file in main branch.
3.  Set up linters.
4.  Added files need to be ignored in gitignore
5.  Created a new branch according to the GitHub flow rules.
6.  Added 2 files in new branch with seperate commits.
7.  Added html file that includes a header with the text "Hello Microverse!.
8. Added css file that include styles for header.

### CODE REVIEW

**Comment by vikas924** [comment]
Hello there,

In this project, I have set up a "Hello world" repository. My tasks include:

1.  Documented a professional README.md.
2.  Included a .gitignore file in main branch.
3.  Set up linters.
4.  Added files need to be ignored in gitignore
5.  Created a new branch according to the GitHub flow rules.
6.  Added 2 files in new branch with seperate commits.
7.  Added html file that includes a header with the text "Hello Microverse!.
8. Added css file that include styles for header.

**Review by ouasamine — CHANGES_REQUESTED** [review]
Hi @vikas924  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Created the HTML & CSS required files ✔️ 
- Included the test file and `node_modules` in the `.gitignore` ✔️ 
- Correct use of GitHub Flow 👏🏽 

## Required Changes ♻️

- [x] Nice work adding a README file to your project, However as per this project's  [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)you are required to use the provided [template](https://github.com/microverseinc/readme-template), also note that you are required to keep all the README template sections except the FAQ which is optional.

__P.S: I can see that you tried to add the README template to your project, but it was committed to your main branch instead of this branch that the PR is based on.
Also please make sure that all the commits that are related to this project's scope are included in the PR because that's the only way a reviewer can review your work, for future projects make sure to create a feature branch and check out to first it before committing any change related to the scope of the activity.__


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] [OPTIONAL] Your PR summary is very good, well done ✔️, however, it sits at the bottom of the page and can easily be missed, note that you can always go back and edit the first comment of the PR and add your summary there, I believe it will look nicer.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,229 @@
...
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```
Comment (Shedrack-Sunday) [inline README.md]: - [x] Nice job adding the changes requested by the previous reviewer. However, kingly add three features this project will likely improve on based on your personal discretion 👍  
- [x] Secondly, please delete the comment on one `173` as it only serves as a guide to you. this keeps your code clean

File: README.md [hunk 2]
```diff
@@ -1 +1,229 @@
...
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Microverse for this exercise.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [x] Here also, please delete the comment on one `193` and `203` as it only serves as a guide to you. this keeps your code clean

File: README.md [hunk 3]
```diff
@@ -1 +1,229 @@
...
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
```
Comment (Shedrack-Sunday) [inline README.md]: -  [x] For this section, you can include possible questions your viewers will be interested in or delete the section completely. 😃

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @vikas924    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements suggested by the previous reviewer,  However, there are a few you need to work on, to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 
  
Required Changes ♻️

- [x] You did a great job with the changes, however, there are a few things I suggested, which I made a detailed review of below 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @vikas924 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉



- Highlights
- Linter checks passing ✔️
- Descriptive PR ✔️
- Good UI design ✔️
# Optional suggestions
The FAQ section in the README file is optional. It would be great if you put some content in it or remove it completely. 😎️
image

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

- [ ] There seems to be conflict in this branch of yours. though i missed it last time out, i strongly suggest that you resolve it before merging 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**